app/furniture/marketplace/cart/delivery_areas/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<%- breadcrumb :edit_marketplace_cart_delivery_area, delivery_area %>
+<%= turbo_frame_tag(dom_id(cart, :delivery_area)) do %>
+  <%= render "form", cart: cart %>
+<%- end %>
```

Comment:
Actually, they are useful because they make the turbo-frames work.

Reply:
Can you add a comment indicating this. I could imagine coming across it without context and being confused.

Reply:
I think it's kind of a learning curve re: Turbo Frames. In most Rails apps, you would expect to have the `#edit/#show/#new/etc` actions for each `Resource`; in Turbo-Streams-land you may not have partials; but as I get better at using Turbo; I'm leaning away from turbo-streams .

Reply:
Cool, yeah. I've noticed "placeholders" for unused actions before. I'm coming from completely bespoke node application development which had very little "templating" as far as class methods, except for specific inheritance cases. But I'm quickly warming up to identifying this pattern.

Reply:
Ah, to clarify these actions are used. I had thought they were not because I had forgotten that turbo-frames do the full request; so my initial comment saying they're not necessary was in error.